src/commands/ext-export.ts

```diff
@@ -19,17 +19,22 @@ module.exports = new Command("ext:export")
   .action(async (options: any) => {
     const projectId = needProjectId(options);
 
-    // Look up the instances that already exist, and add a ^ to their version.
-    const have = (await planner.have(projectId)).map((s) => {
-      if (s.ref) {
-        s.ref.version = `^${s.ref.version}`;
-      }
-      return s;
-    });
+    // Look up the instances that already exist,
+    // add a ^ to their version,
+    // and strip project IDs from the param values.
+    const have = (await planner.have(projectId))
```

Comment:
what's `have`?... 

Somehow reminds me of: https://www.youtube.com/watch?v=HTjqz0_wsZo

Reply:
🤣  

This is copying the pattern we use over in function deploy - we compare what you 'have' deployed on your project now to what to 'want' deployed.